tasks/apply-mapping/apply-mapping.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: apply-mapping
   labels:
-    app.kubernetes.io/version: "1.5.1"
+    app.kubernetes.io/version: "1.6.0"
```

Comment:
No-op: One could argue that it's a breaking change. But I think this is ok. My expectation is that users won't notice or mind (except those that actually expressed the need for build-time timestamps).

Reply:
I left it as is since I didn't add or remove a task parameter, but I don't really have a strong opinion on it

Reply:
Yep, I think it's fine. I was just thinking out loud :)